src/main/java/org/gridsuite/modification/server/repositories/ModificationRepository.java

```diff
@@ -27,6 +27,9 @@ public interface ModificationRepository extends JpaRepository<ModificationEntity
     @Query(value = "SELECT new ModificationEntity(m.id, m.type, m.date, m.stashed, m.messageType, m.messageValues) FROM ModificationEntity m WHERE m.group.id = ?1 order by m.modificationsOrder")
     List<ModificationEntity> findAllBaseByGroupId(UUID uuid);
 
+    @Query(value = "SELECT m FROM ModificationEntity m WHERE m.id IN :uuids order by m.modificationsOrder")
+    List<ModificationEntity> findAllByIdIn(List<UUID> uuids);
```

Comment:
why not using directly the form `findAllByIdInOrderByModificationsOrder` instead of this JPQL query ?